.github/workflows/pr_validation.yaml

```diff
@@ -0,0 +1,71 @@
+name: PR checks
```

Comment:
WDYT about adding the generated name as for e2e (prior to this PR) to include the branch name?